### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by hardlevel** [fgsfdsfgs/max_vita#7]
Hi! I did all the steps, installed the plugins, pkg and everything else, but I got crash on startup, on the image follows my maxpayne folder, but I don`t know where is my crash dump, can you help me?
![Screenshot_4](https://user-images.githubusercontent.com/23259881/106337802-e7d09d80-6270-11eb-8030-88486c6ab291.png)

**Comment by fgsfdsfgs** [comment]
Check the 4 closed issues, see if any of the solutions work for you.

**Comment by hardlevel** [comment]
Hi thanks for the answer, but the shader file must be on the maxpayne folder? Mine is on data folder on ur0

**Comment by fgsfdsfgs** [comment]
No, it must be in `ur0:/data`. If you're just now installing it for Max Payne, check the "note 2" at the very bottom of the shader compiler installation guide.

**Comment by hardlevel** [comment]
hi, I recreated the folder and moved the files as mentioned on note2, but when I did it, my folder maxpayne in ux0 was gone, and I had to move it again and install the vpk, still not working

**Comment by fgsfdsfgs** [comment]
Try [the new release](https://github.com/fgsfdsfgs/max_vita/releases/tag/r3), see if it shows an error.
